hooks/playbooks/manila_create_default_resources.yml

```diff
@@ -8,7 +8,31 @@
     extra_specs:
       snapshot_support: "True"
       create_share_from_snapshot_support: "True"
+    manila_provider_network_name: storage
+    manila_provider_network_vlan: 21
+    manila_provider_network_start: 172.18.0.150
+    manila_provider_network_end: 172.18.0.200
+    manila_provider_network_range: 172.18.0.0/24
   tasks:
+    - name: Create Manila provider network with Neutron for instance to access Manila
+      environment:
+        KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+        PATH: "{{ cifmw_path }}"
+      ansible.builtin.command: |
+        oc -n {{ namespace }} exec -it pod/openstackclient -- {{ item }}
+      loop:
+        - "openstack network create {{ manila_provider_network_name }} --share --provider-network-type vlan --provider-physical-network datacentre --provider-segment {{ manila_provider_network_vlan }}"
+        - "openstack subnet create --allocation-pool start={{ manila_provider_network_start }},end={{ manila_provider_network_end }} --dhcp --network {{ manila_provider_network_name }} --subnet-range {{ manila_provider_network_range }} --gateway none {{ manila_provider_network_name }}-subnet"
+      register: _manila_provider_network_creation
+      failed_when: >-
+        ( _manila_provider_network_creation.rc | int ) != 0
+      when:
```

Comment:
A small suggestion: maybe we can move the `when`s to just after the name so that it is easier to read? (Not sure if we are following this convention everywhere, but I remember Cedric mentioning this in some cases)

Reply:
Yes, I can do that.

Reply:
The `when`s have been moved.